Source/JavaScriptCore/runtime/DataView.h

```diff
@@ -34,13 +34,8 @@ class DataView final : public ArrayBufferView {
 public:
     JS_EXPORT_PRIVATE static Ref<DataView> create(RefPtr<ArrayBuffer>&&, size_t byteOffset, size_t length);
     static Ref<DataView> create(RefPtr<ArrayBuffer>&&);
-    
-    TypedArrayType getType() const final
-    {
-        return TypeDataView;
-    }
 
-    JSArrayBufferView* wrap(JSGlobalObject*, JSGlobalObject*) final;
+    JSArrayBufferView* wrapImpl(JSGlobalObject*, JSGlobalObject*);
```

Comment:
I’m personally not all that big a fan of the "Impl" suffix as a way to name things like this, but it’s consistent with use elsewhere. Also seems to me that the two arguments need names. It’s not obvious which means which when they are both arguments of the same type.

Reply:
Added parameter names. For now, as a naming convention aligned with the other places, using `wrapImpl`.